generators/csharp/sdk/src/SdkGeneratorContext.ts

```diff
@@ -295,7 +295,7 @@ export class SdkGeneratorContext extends AbstractCsharpGeneratorContext<SdkCusto
         return Object.values(AsIsFiles.Test.Utils);
     }
 
-    public getExampleEndpointCallOrThrow(endpoint: HttpEndpoint): ExampleEndpointCall {
+    public getExampleEndpointCallIfExists(endpoint: HttpEndpoint): ExampleEndpointCall {
```

Comment:
The return should be `ExampleEndpointCall | undefined`, and then use `return undefined` instead of `return undefined as unknown as ExampleEndpointCall`